src/Symfony/Component/String/UnicodeString.php

```diff
@@ -84,6 +84,13 @@ public function chunk(int $length = 1): array
         return $chunks;
     }
 
+    public function codePointsAt(int $offset): array
```

Comment:
could be moved to AbstractUnicodeString with same reasoning as bytesAt on AbstractString